main.go

```diff
@@ -22,59 +29,178 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Collect Go Track1 function declarations that returns
-	pwd, _ := os.Getwd()
 	for _, pkg := range pkgs {
-		for _, f := range pkg.Syntax {
-			ast.Inspect(f, func(node ast.Node) bool {
-				asnmt, ok := node.(*ast.AssignStmt)
-				if !ok {
-					return true
-				}
-				if len(asnmt.Lhs) != 2 || len(asnmt.Rhs) != 1 {
-					return true
-				}
+		trackOneSDKScan(pkg, pwd)
+		pandoraSDKScan(pkg, pwd)
+	}
+}
 
-				ret1, ok := asnmt.Lhs[0].(*ast.Ident)
-				if !ok {
-					return true
-				}
-				if ret1.Name != "_" {
-					return true
-				}
+func trackOneSDKScan(pkg *packages.Package, rootPath string) {
```

Comment:
```suggestion
func trackOneSDKScan(pkg *packages.Package, pwd string) {
```

Can we rename it to be `pwd` to make it clear?